lib/ClangImporter/ImportDecl.cpp

```diff
@@ -6618,15 +6627,19 @@ SwiftDeclConverter::importSwiftNewtype(const clang::TypedefNameDecl *decl,
     if (unlabeledCtor)
       options |= MakeStructRawValuedFlags::MakeUnlabeledValueInit;
 
-    synthesizer.makeStructRawValued(structDecl, storedUnderlyingType,
```

Comment:
I actually have mixed feelings about this. Now the callers must remember to add the protocols after making a call to this API. Previously, it happend automatically is it was harder to misuse this. I think I prefer the old code in this case.

Reply:
Agreed, let's not move the protocol conformances out of the method.

Reply:
Sounds good, I was on the fence about this change too and was already thinking about reverting it.